docs/vercel.json

```diff
@@ -1,13 +1,14 @@
 {
-  "version": 2,
-  "name": "primer-viewcomponents",
   "builds": [
     {
       "src": "package.json",
-      "use": "@now/static-build",
+      "use": "@vercel/static-build",
       "config": {"distDir": "public"}
     }
   ],
   "rewrites": [{"source": "/view-components(/.*)?", "destination": "/$1"}],
-  "redirects": [{"source": "/", "destination": "/view-components"}]
+  "redirects": [{"source": "/", "destination": "/view-components"}],
+  "github": {
+    "enabled": false
+  }
```

Comment:
what is this?

Reply:
It stops the Vercel bot from deploying (the thing that makes the comments on PRs) 6ee245630b5b0138997ee9b21e2cf1a132301fe3